pulp_rpm/tests/functional/utils.py

```diff
@@ -272,3 +274,10 @@ def progress_reports(task_href):
         return task.progress_reports
 
     return []
+
+
+def get_package_repo_path(package):
+    """Get package repo path with directory structure."""
```

Comment:
Args/Returns  description is missing here

Reply:
And maybe the name of the arg can be more descriptive.

The only reason I noticed that the docstring is not full is because I expected `package` to be a Package object but it turned out to be a string. Becasue it's not a package, it's a path.